Frontend/implementations/typescript/src/player.ts

```diff
@@ -22,7 +22,30 @@ document.body.onload = function() {
 
 	const application = new Application({
 		stream,
-		onColorModeChanged: (isLightMode) => PixelStreamingApplicationStyles.setColorMode(isLightMode)
+		onColorModeChanged: (isLightMode) => PixelStreamingApplicationStyles.setColorMode(isLightMode),
+		settingsPanelConfig: {
+			isEnabled: true,
+			settingVisibility: {
+			  StreamerId: false,
+			  ss: false,
+			  TimeoutIfIdle: false,
+			  ForceTURN: false,
+			  LightMode: false,
+			  UseMic: false,
+			  MatchViewportRes: false,
+			  HoveringMouse: false,
+			  AFKTimeout: false,
+			  AFKCountdown: false,
+			  MaxReconnectAttempts: false,
+			  StartVideoMuted: false
+			},
+			sectionVisibility: {
+			  Encoder: false,
+			  WebRTC: false,
+			  UI: false,
+			  Input: false
+			}
+		  }
```

Comment:
The typescript example shouldn't need any of this new config, right?

Reply:
Ah, bit of dev work seeping in. Fixed!